src/imio/actionspanel/locales/es/LC_MESSAGES/imio.actionspanel.po

```diff
@@ -0,0 +1,128 @@
+# Translation of imio.actionspanel.pot to Spanish
+# Leonardo J. Caballero G. <leonardocaballero@gmail.com>, 2020.
+msgid ""
+msgstr ""
+"Project-Id-Version: imio.actionspanel\n"
+"POT-Creation-Date: 2016-08-24 08:23+0000\n"
+"PO-Revision-Date: 2020-04-27 00:57-0400\n"
+"Last-Translator: Leonardo J. Caballero G. <leonardocaballero@gmail.com>\n"
+"Language-Team: ES <LL@li.org>\n"
+"Language: es\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=utf-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"X-Generator: Virtaal 0.7.1\n"
+"Language-Code: es\n"
+"Language-Name: Spanish\n"
+"Preferred-Encodings: utf-8 latin1\n"
+"Domain: imio.actionspanel\n"
+"X-Is-Fallback-For: es-ar es-bo es-cl es-co es-cr es-do es-ec es-es es-sv es-gt es-hn es-mx es-ni es-pa es-py es-pe es-pr es-us es-uy es-ve\n"
+
+#. Default: "Item state changed."
+#: ./browser/views.py:502
+msgid "%s_done_descr"
+msgstr "Estado del elemento cambiado."
+
+#: ./browser/registry.py:16
+msgid "Define for each object transition the optional corresponding view"
+msgstr ""
+"Definir para cada transición de objeto la vista correspondiente opcional"
```

Comment:
Hi @macagua 

this seems a typo error?

Thank you for this PR,

Gauthier

Reply:
Fixed on commit [37966a0]